R/deprecate.R

```diff
@@ -174,12 +196,34 @@ deprecate_warn0 <- function(msg,
   env_poke(deprecation_env, id, Sys.time())
 
   footer <- function(...) {
+    footer <- NULL
+
+    if (!direct) {
+      top <- topenv(user_env)
+
+      if (is_namespace(top)) {
+        pkg <- ns_env_name(top)
+        footer <- c(
+          footer,
+          "i" = cli::format_inline(
+            "The deprecated feature was likely used in the {.pkg {pkg}} package."
+          ),
+          " " = cli::format_inline(
+            "Please report the issue to the authors."
```

Comment:
This might be scope create, but is it worth doing some lightweight parsing of `packageDescription("purrr")$BugReports` and adding a clickable link if it starts with `https://`?

Reply:
yup I'll take a look, and also do it in `abort(.internal = TRUE)` which is a feature @DavisVaughan also requested.